src/main/java/org/jenkins/plugins/lockableresources/LockableResourcesManager.java

```diff
@@ -741,7 +745,8 @@ public boolean configure(StaplerRequest req, JSONObject json) throws FormExcepti
   public synchronized Set<LockableResource> checkResourcesAvailability(
       List<LockableResourcesStruct> requiredResourcesList,
       @Nullable PrintStream logger,
-      @Nullable List<String> lockedResourcesAboutToBeUnlocked) {
+      @Nullable List<String> lockedResourcesAboutToBeUnlocked,
+      boolean skipIfLocked) {
```

Comment:
Please provide a compatibility function with the old signature - You can then keep the code paths were the new functionality isn't used the same as before.

Reply:
Done.